arangod/Aql/AstNode.cpp

```diff
@@ -376,6 +383,57 @@ static bool IsEmptyString(char const* p, size_t length) {
   return true;
 }
 
+/*static*/ std::string AstNode::encodeDataSourceType(
+    char* const name, size_t size, AstNode::DataSourceType type
```

Comment:
AFAIK, it should be `char const*`, because `char* const` has a different meaning:
- `char const*` means that the data pointed to is const
- `char* const` means the pointer is const

Reply:
yes, misspelling